src/containers/BookForm.js

```diff
@@ -0,0 +1,49 @@
+import React from "react"
+import Form from "react-bootstrap/Form"
+import { Button } from "react-bootstrap"
+import { useForm } from "react-hook-form"
+
+const BookForm = () => {
+  const { register, handleSubmit, watch, errors } = useForm()
+  const onSubmit = data => {
+    console.log(data)
```

Comment:
- [ ] We need to make our code clean before merging this milestone. Therefore we need to remove all console logs. We need to remove this console log here